### PR TITLE
Env support

### PR DESCRIPTION
**Pull request by XVincentX**

### CODE REVIEW

**Review by DrMegavolt — APPROVED** [review]
LGTM

File: docs/configuration/index.md [hunk 1]
```diff
@@ -18,12 +18,48 @@ Configuration is divided into different levels:
 
 The levels allow you to configure and manage Express Gateway without having to concern yourself with details that may not be relevant to you as a user, operator, administrator or developer. The lower the level, the higher the complexity.
 
+## Environment variables in configuration files
+
+You can use environment variables in configuration files; they follow the same syntax of `docker-compose.yml` files.
```
Comment (altsang) [inline docs/configuration/index.md]: can we do [Docker Compose YAML](https://docs.docker.com/compose/environment-variables/)

File: docs/faq.md [hunk 2]
```diff
@@ -33,6 +33,11 @@ Express Gateway currently supports Redis as both its primary and distributed dat
 
 Express Gateway config files are easy to modify and maintain as flat files in the `/config` directory.
 
+### Can I use environment variables in config files?
+
+Yes, you can! We have borrowed the same syntax that `docker-compose.yml` files are using. Check out the relative
```
Comment (altsang) [inline docs/faq.md]: [Docker Compose YAML](https://docs.docker.com/compose/environment-variables/)

**Review by altsang — CHANGES_REQUESTED** [review]

**Comment by XVincentX** [comment]
@altsang Done!

**Pull request merged**